script.adzapper/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.adzapper" name="LiveTV Ad-Zapper" version="1.0.1" provider-name="pOpY">
+	<requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.python.script" library="default.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Rezap to current channel after some time (advertisment)</summary>
+		<description lang="en_GB">Add advertisment skipping to your Kodi system, allowing the user to define a time to switch back to the current channel. During this time the user can switch channel without thinking to switch back manually.</description>
+		<disclaimer lang="en_GB"></disclaimer>
```

Comment:
If disclaimer is set you need to provide some text, otherwise just remove from addon.xml